### PR TITLE
Add tool to download the attachments from sqlite3 using wget instead of trac-admin

### PR DESCRIPTION
**Pull request by aerostitch**
It's the equivalent from the mysql one.
Allows to do the download from a remote server with just a copy of the sqlite3 db.

### CODE REVIEW

File: tools/download-trac-attachment-sqlite3.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# make sure the output dir is clean
+# rm -rf ${OUTPUT_DIR}
+
+
+# walk through the output
+IFS=$'\n'
+for line in ${output}; do
+  ticket=$(echo $line | cut -d '|' -f 1)
+  file=$(echo $line | cut -d '|' -f 2-)
+  if [[ $ticket == id ]]; then
```
Comment (mavam) [inline tools/download-trac-attachment-sqlite3.sh]: Just a nit: I'm a big fan of POSIX shell scripts. If you change this to `[ "$ticket" = "id" ]` it'll be more portable.

File: tools/download-trac-attachment-sqlite3.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    continue
+  fi
+
+  mkdir -p ${OUTPUT_DIR}/${ticket}
```
Comment (mavam) [inline tools/download-trac-attachment-sqlite3.sh]: The argument to `mkdir` should be wrapped into double-quotes. You do this already below. :-)

**Review by mavam — CHANGES_REQUESTED** [review]
Thanks for the contribution!

I only have some minor syntactic change requests.

**Comment by aerostitch** [comment]
Thanks a lot for the review (and the other merges)!
I just pushed the new version with the changes.

**Review by mavam — APPROVED** [review]

**Comment by mavam** [comment]
And thank you for all the work of putting together these great PRs! ❤️

**Pull request merged**